funnel/views/update.py

```diff
@@ -184,6 +184,22 @@ def delete(self):
             cancel_url=self.obj.url_for(),
         )
 
+    @route('next')  # NOQA: A003
+    def next(self):  # NOQA: A003
+        nextobj = self.obj.getnext()
+        if nextobj:
+            return redirect(nextobj.url_for())
+        else:
+            return redirect(self.obj.project.url_for())
+
+    @route('prev')
+    def prev(self):
+        prevobj = self.obj.getprev()
+        if prevobj:
+            return redirect(prevobj.url_for())
+        else:
+            return redirect(self.obj.project.url_for())
```

Comment:
This routes are not required, as we're rendering with the actual URL of the next and previous update.